CMakeLists.txt

```diff
@@ -119,3 +102,12 @@ install(FILES
     DESTINATION "${CMAKE_INSTALL_MANDIR}/man1"
     COMPONENT Runtime
 )
+
+MACRO(install_symlink FILEPATH SYMPATH)
+    install(CODE "message(\"-- Creating symlink: ${SYMPATH} -> ${FILEPATH}\")")
+    install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
+      \"${FILEPATH}\" \"${SYMPATH}\")")
+ENDMACRO(install_symlink)
+
+install_symlink( lxqt-sudo "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${LINK_LXSU}"   )
+install_symlink( lxqt-sudo "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${LINK_LXSUDO}" )
```

Comment:
Will not work, FTBFS

```
failed to create symbolic link '/usr/bin/lxsu' because existing path cannot be removed: Permission denied
-- Creating symlink: /usr/bin/lxsudo -> lxqt-sudo
failed to create symbolic link '/usr/bin/lxsudo' because existing path cannot be removed: Permission denied
make[1]: Leaving directory '/home/agaida/work/code/pkg-main/lxqt-sudo/snapshot/obj-x86_64-linux-gnu'
   dh_install -O--buildsystem=cmake
dh_install: Cannot find (any matches for) "usr/bin/lxsu" (tried in ., debian/tmp)

dh_install: lxqt-sudo missing files: usr/bin/lxsu
dh_install: Cannot find (any matches for) "usr/bin/lxsudo" (tried in ., debian/tmp)
``` 
The links has to be created in the current binary dir and after installed in the targeted place - so in current form not good to merge,